code/datums/components/food/decomposition.dm

```diff
@@ -73,7 +73,7 @@
 
 	var/turf/open/open_turf = food.loc
 
-	if(!istype(open_turf) || istype(open_turf, /turf/open/lava) || istype(open_turf, /turf/open/floor/plating/asteroid/basalt)) //Are we actually in a valid open turf?
+	if(!istype(open_turf) || istype(open_turf, /turf/open/lava) || istype(open_turf, /turf/open/floor/plating/asteroid)) //Are we actually in a valid open turf?
```

Comment:
water too?

Reply:
This check sucks why can’t we just make it a list like we do for elevated structures

Reply:
The elevated structures list said only objects so I assumed that meant I couldn't make a list like that one for turfs

Are you able to make them for turfs? If so that just makes me feel stupid for not realizing it and using that instead.